troposphere/static/js/components/NotFoundPage.react.js

```diff
@@ -10,7 +10,7 @@ export default React.createClass({
 
         return (
         <div className="container">
-            <h2>Page Unavailable</h2>
+            <h1 className="t-display">Page Unavailable</h1>
```

Comment:
Can you document this somewhere? I do not know what a "t-display" is, or the convention of a leading "t".